README.md

```diff
@@ -90,6 +90,41 @@ in node:
 
 Feel free to [try immutad●t on runkit](https://npm.runkit.com/immutadot).
 
+## Dot notation
+
+immutad●t brings a few improvements to the classic dot notation:
+
+### Slice notation
+
+The slice notation let you go trough arrays to apply operations without having to map arrays at each level of imbrication.
```

Comment:
I would prefer "The slice notation lets you iterate over arrays...", what do you think ?